src/test/java/seedu/address/logic/commands/DeleteCommandTest.java

```diff
@@ -35,19 +40,74 @@ public void execute_deleteValidName_success() {
         assertCommandSuccess(deleteCommand, model, expectedMessage, expectedModel);
     }
 
+    @Test
```

Comment:
Should there also be tests to test for names not in the `Model`? Or was this intended?